<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Dépenses</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="Impression">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Afficher Dépense</string>
     </attribute>
     <widget class="QTableView" name="table_depense">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>551</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_is">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>identifiant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="tri_date">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>220</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par date</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="tri_identifiant">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>220</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par identifiant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="tri_type">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>220</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par type</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ajouter Dépense</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>731</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>montant</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ma">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ia">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>identifiant</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_sa">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Salaire</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_ia">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Impot</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_feta">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>60</y>
         <width>131</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Frais d'entretien</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_ra">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>60</y>
         <width>151</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Remboursement</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_feqa">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>60</y>
         <width>151</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Frais d'equipement</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_a">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modifier Dépense</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>montant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mm">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_im">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>identifiant</string>
      </property>
     </widget>
     <widget class="QTableView" name="t_depense">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>371</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_sm">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Salaire</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_im">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Impot</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_fetm">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Frais d'entretien</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_rm">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>150</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Remboursement</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_feqm">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>150</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Frais d'équipement</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>250</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_m">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Calculatrice</string>
     </attribute>
     <widget class="QLineEdit" name="Display">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>551</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{

background-color:gray;
border: 1px solid gray;
color: #ffffff;

}
</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="clear">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_0">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="change_signe">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="soustraction">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="egal">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>190</y>
        <width>93</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#AFEEEE;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addition">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="multiplication">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="division">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

background-color:#F0F8FF;
border: 1px solid gray;
padding: 5px;

}
QPushButton:pressed{

background-color:#48D1CC;
border: 1px solid gray;
padding: 5px;

}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Text</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>16</y>
        <width>701</width>
        <height>271</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/1x/Fichier.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Historique</string>
     </attribute>
     <widget class="QTableView" name="table_ha">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>851</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="table_hm">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>851</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Historique des modifications</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>-4</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Historique des ajouts</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionSortir"/>
    <addaction name="actionImprimer"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouveau"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionSortir"/>
   <addaction name="actionImprimer"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouveau">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/nouveau.png</normaloff>:/nouveau.png</iconset>
   </property>
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/ouvrir.png</normaloff>:/ouvrir.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionSortir">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/fermer.jpg</normaloff>:/fermer.jpg</iconset>
   </property>
   <property name="text">
    <string>Sortir</string>
   </property>
  </action>
  <action name="actionImprimer">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/imprimer.webp</normaloff>:/imprimer.webp</iconset>
   </property>
   <property name="text">
    <string>Imprimer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
